README.rst

```diff
@@ -1,25 +1,23 @@
-django-menu
+Django-menu
 -----------
 
 BSD-licensed menu tools for Django, built by Ross Poulton <http://www.rossp.org>
 
-django-menu provides a basic structure for you to build multiple navigation 
-menus for your website, such as the header menubar. These menus can be easily 
-maintained by staff using the Django administration without any knowledge 
-of HTML or Django internals.
+Django-menu provides a basic structure for you to build multiple navigation menus for your websites, like the header menubar. These menus can be easily 
+maintained by the staff using the Django administration without any knowledge of HTML or Django internals.
```

Comment:
It looks like the author chose to stay between 80 or 120 characters per line; this is supported by Markdown, is not reflected in the rendered text and is a valid choice.
Keeping line lengths short increases readability in text editors and when viewing text files in the terminal.
Consider keeping this section in multiple, shorter lines